src/researches/findKeywordInPageTitle.js

```diff
@@ -1,24 +1,42 @@
 /** @module analyses/findKeywordInPageTitle */
-
-var wordMatch = require( "../stringProcessing/matchTextWithWord.js" );
-
-var escapeRegExp = require( "lodash/escapeRegExp" );
+const functionWords = require( "../helpers/getFunctionWords" )();
+const wordMatch = require( "../stringProcessing/matchTextWithWord" );
+const escapeRegExp = require( "lodash/escapeRegExp" );
+const stripSpaces = require( "../stringProcessing/stripSpaces" );
+const removePunctuation = require( "../stringProcessing/removePunctuation" );
+const createRegexFromArray = require( "../stringProcessing/createRegexFromArray" );
+const getLanguage = require( "../helpers/getLanguage" );
+const isUndefined = require( "lodash/isUndefined" );
 
 /**
  * Counts the occurrences of the keyword in the pagetitle. Returns the number of matches
  * and the position of the keyword.
  *
- * @param {object} paper The paper containing title and keyword.
- * @returns {object} result with the matches and position.
+ * @param {Object} paper The paper containing title and keyword.
+ *
+ * @returns {Object} Result with the matches and position.
  */
 
 module.exports = function( paper ) {
-	var title = paper.getTitle();
-	var keyword = escapeRegExp( paper.getKeyword() );
-	var locale = paper.getLocale();
-	var result = { matches: 0, position: -1 };
+	let title = removePunctuation( paper.getTitle() );
+	let keyword = escapeRegExp( paper.getKeyword() );
+	const locale = paper.getLocale();
```

Comment:
Seeing as this variable is only used once on line 26, I'd suggest removing this line and just call `paper.getLocale()` on line 26 as the third parameter.

Reply:
also line 43